tests/test_workspace.py

```diff
@@ -453,6 +453,7 @@ def test_combine_workspace_incompatible_poi(workspace_factory, join):
 def test_combine_workspace_diff_version(workspace_factory, join):
     ws = workspace_factory()
     ws.version = '1.0.0'
+    ws['version'] = '1.0.0'
```

Comment:
I'm probably missing the point of this test, but if you put a `breakpoint` just after `ws = workspace_factory()`

```
(Pdb) ws.version
'1.0.0'
(Pdb) ws["version"]
'1.0.0'
```

so is the reason to explicitly set this a means to solidify the versions that will be used in the future?

Reply:
This is just a test to enforce absolutely different versions.